api/src/main/java/com/codeforcommunity/dto/report/ReportWithLibrary.java

```diff
@@ -29,7 +30,8 @@
   private Boolean hasSufficientTraining;
   private String teacherSupport;
   private String parentSupport;
-  private JsonNode timetable;
+  private JsonNode checkInTimetable;
+  private JsonNode checkOutTimetable;
```

Comment:
Does this require changing the request on the frontend?

Reply:
Yes it does. I will add it to Dev Docs so the frontend team can have the most recent request fields

Reply:
Wait but then this will just break the frontend, 😬 shouldn't this be accompanied by another PR that handles this breaking change?